### PR TITLE
LPS-159627 Remove autocomplete from Color Picker

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/common/components/ColorPicker/ColorPicker.test.js [hunk 1]
```diff
@@ -113,7 +114,7 @@ describe('ColorPicker', () => {
 
 		fireEvent.click(getByTitle('clear-selection'));
 
-		expect(baseElement.querySelector('input').value).toBe('#ABCABC');
+		expect(baseElement.querySelector('input')).toHaveValue('#ABCABC');
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/test/page_editor/common/components/ColorPicker/ColorPicker.test.js:115]: 😍

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/common/components/ColorPicker/ColorPicker.test.js [hunk 2]
```diff
@@ -81,9 +81,7 @@ const renderColorPicker = ({
 	);
 
 const onTypeValue = (input, value) => {
-	fireEvent.change(input, {
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/test/page_editor/common/components/ColorPicker/ColorPicker.test.js:83]: 💃

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/common/components/ColorPicker/ColorPicker.test.js [hunk 3]
```diff
@@ -128,7 +126,7 @@ describe('ColorPicker', () => {
 		it('shows action buttons when the color picker is clicked', async () => {
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/test/page_editor/common/components/ColorPicker/ColorPicker.test.js:126]: There are some async in this test suite that are not needed, could you review it and remove it?

Comment (veroglez) [reply modules/apps/layout/layout-content-page-editor-web/test/page_editor/common/components/ColorPicker/ColorPicker.test.js]: Done!

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/ColorPicker/ColorPicker.js [hunk 4]
```diff
@@ -330,82 +280,22 @@ export function ColorPicker({
...
-										</ClayAutocomplete.DropDown>
-									</ClayAutocomplete>
-								</FocusScope>
+								<ClayInput
+									aria-invalid={error.label}
+									aria-label={field.label}
+									className="page-editor__color-picker__input"
+									id={inputId}
+									onBlur={onBlurInput}
+									onChange={onChangeInput}
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/ColorPicker/ColorPicker.js:289]: I think we should also implement the behaviour we have in other input text, that when you click enter the value is saved, the same way as we save it when the blur event is triggered

**Comment by victorg1991** [comment]
Only one minor change and this should be ready to go 🥳

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**